src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -1716,33 +1716,27 @@ public void updateTriggerConfigURL() {
         }
         triggerInformationAction.setErrorMessage("");
         try {
-            if (isAnyServer()) {
-                triggerInformationAction.setErrorMessage("Dynamic trigger configuration needs "
```

Comment:
I can't remember why it needed this? And I get paranoid about it since I spent months tweaking the multi server configurations to not break.

Reply:
It was needed because very long time ago serverName in `GerritDynamicUrlProcessor.fetch` was really used.